src/components/InterviewScheduleModal.tsx

```
@@ -0,0 +1,411 @@
+                    />
+                    <span className='text-sm text-gray-700'>{topic}</span>
+                  </label>
+                ))}
+              </div>
+            </div>
+
+            {/* Timezone Display */}
+            <div>
+              <label className='block text-sm font-medium text-dark mb-2'>
+                Your Timezone
+              </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)